readthedocs/core/utils/__init__.py

```diff
@@ -172,7 +172,7 @@ def trigger_build(project, version=None, record=True, force=False):
         # Current project is skipped
         return None
 
-    return update_docs_task.apply_async()
+    return (update_docs_task.apply_async(), update_docs_task)
```

Comment:
The docstring needs to be updated accordingly.

Reply:
Yeah. Missed it. Also is the `AsyncResult` promise necessary? I don't find any use of returning it from the function.

Reply:
It's not used, I think.

It returns the async result since it's what it creates and could be useful to extend this from outside.

Reply:
Modify the test and update the docstring to reflect your changes.

Reply:
@humitos Sorry but I still can't figure out what to modify in [test_core_utils](https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/rtd_tests/tests/test_core_utils.py) or in other tests since they do not utilize the `build_pk`

Reply:
@humitos @agjohnson Ping. Need some help with the tests.

> I still can't figure out what to modify in `test_core_utils` or in other tests since they do not utilize the `build_pk`.